### PR TITLE
UI page for event details

### PR DESCRIPTION
**Pull request by jean-neiverth**
- Events listed are now clickable to lead to the details page.
- The event data shown in the event/eventID are the actual data of that id
- The ticket data, otherwise, is a static example list of owners 

https://www.loom.com/share/f3d10da04eb4438db2c67df5a32ed3bf?sid=ed4a301e-021b-47ce-9a08-0c20e7876229

### CODE REVIEW

File: src/app/event/components.tsx
Comment (luizakp) [inline src/app/event/components.tsx:1]: Nice

File: src/app/events/page.tsx [hunk 1]
```diff
@@ -42,28 +47,42 @@ export default function Events() {
...
+        <h2 className="text-xs text-gray-700 mb-4">
+          Brief description: {parsedData.briefDescription}
+        </h2>
+        <h2 className="text-xs text-gray-700 mb-4">
+          Full description: {parsedData.fullDescription}
+        </h2>
       </div>
-    </div>
-
+      {/* </div> */}
```
Comment (luizakp) [inline src/app/events/page.tsx]: 👀

File: src/lib/gqlEasAttestation/query.ts [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+      attester
+      recipient
+      refUID
+      revocable
+      revocationTime
+      expirationTime
+      data
+      decodedDataJson
+    }
   }
```
Comment (luizakp) [inline src/lib/gqlEasAttestation/query.ts]: 🔝

**Review by luizakp — APPROVED** [review]
Nice work! this is function. I have some suggestion tho, will send on another PR

**Review by luizakp — APPROVED** [review]

**Comment by luizakp** [comment]
think this is not working on mobile tho 👀

File: src/app/event/components.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import type { ReactNode } from "react";
+import { MaterialSymbol } from "react-material-symbols";
+
+export const EventInfoContainer = ({ children }: { children: ReactNode }) => {
+  return (
+    <div className="w-1150px block py-4 px-8 border-2 rounded-3xl mb-10">
```
Comment (luizakp) [inline src/app/event/components.tsx]: this is why it's not working on mobile

Comment (luizakp) [inline src/app/event/components.tsx]: instead of mb-10, prefer using the gap-y on the parent component

File: src/app/event/components.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      <span>Published</span>
+    </div>
+  );
+};
+
+export const TicketInfoContainer = ({ children }: { children: ReactNode }) => {
+  return (
+    <div className="w-1150px py-12 px-12 border-2 rounded-3xl">{children}</div>
+  );
+};
```
Comment (luizakp) [inline src/app/event/components.tsx:25]: why make this a component? do we think this will repeat?

Comment (jean-neiverth) [reply src/app/event/components.tsx:25]: Just to make more clean / readable

**Pull request merged**